docs/models/ipam/asn.md

```diff
@@ -1,14 +1,14 @@
 # ASNs
 
-An Autonomous System Number (ASN) is a numeric identifier used in the BGP protocol to identify which [autonomous system](https://en.wikipedia.org/wiki/Autonomous_system_%28Internet%29) a particular prefix is originating and transiting through. NetBox support both 32- and 64- ASNs.
+An Autonomous System Number (ASN) is a numeric identifier used in the BGP protocol to identify which [autonomous system](https://en.wikipedia.org/wiki/Autonomous_system_%28Internet%29) a particular prefix is originating and transiting through. NetBox support both 16- and 32-bit ASNs.
```

Comment:
While you are at it, how about fixing this invalid sentence?

Right now it says in essence "AS a particular prefix is originating" which means the prefix would be originating the AS.  
Obviously this is the other way around, the AS originates a prefix.

Additionally origination and transiting are mutually exclusive imho so change the "and" to "or".

```suggestion
An Autonomous System Number (ASN) is a numeric identifier used in the BGP protocol to identify which [autonomous system](https://en.wikipedia.org/wiki/Autonomous_system_%28Internet%29) a particular prefix is originated from or transiting through. NetBox support both 16- and 32-bit ASNs.
```

Reply:
OK